ide-support/revliburl.livecodescript

```diff
@@ -73,6 +80,49 @@ on libraryStack
    initialiseCustomProps
 end libraryStack
 ------------------------------------------------
+-- CW-2016-06-11: [[ External driver support ]] Add function to select external driver for use with libUrl.
+## Set driver to use for libUrl
+on libUrlSetDriver pDriver
```

Comment:
@techstrategies I'm contemplating how to generalise the inclusion of a library, back script, front script or behavior either with a supporting external. The concept of an inclusion in the standalone builder is broken up into library, external, lcb extension when we really need an inclusion to be capable of having one or more of those.